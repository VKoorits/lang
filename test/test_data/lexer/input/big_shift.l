if a > 5:
		print "big value"
