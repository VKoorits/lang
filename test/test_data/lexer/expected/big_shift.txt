ERROR: very big shift 2:
  		print "big value"
